source/options-storage.ts

```diff
@@ -36,14 +36,15 @@ export const renamedFeatures = new Map<string, string>([
 	['navigate-pages-with-arrow-keys', 'pagination-hotkey'],
 	['list-pr-for-branch', 'list-prs-for-branch'],
 	['quick-label-hiding', 'quick-label-removal'],
-	['next-scheduled-github-action', 'scheduled-and-manual-workflow-indicators'],
+	['next-scheduled-github-action', 'github-actions-indicators'],
 	['raw-file-link', 'more-file-links'],
 	['conversation-filters', 'more-conversation-filters'],
 	['quick-pr-diff-options', 'one-click-diff-options'],
 	['quick-review-buttons', 'one-click-review-submission'],
 	['wait-for-build', 'wait-for-checks'],
 	['pull-request-hotkey', 'pull-request-hotkeys'],
 	['first-published-tag-for-merged-pr', 'closing-remarks'],
+	['scheduled-and-manual-workflow-indicators', 'github-actions-indicators'],
```

Comment:
@yakov116 naming ideas? I think we should use "GitHub actions" and another word. I'm tired of not being able to use the feature name in commits because it's so long and nothing else fits

Reply:
Sorry forgot to respond
Maybe `scheduled-workflow-indicators`?

Reply:
Heheh too late now 😅 the change has been made

Reply:
No worries! I saw it on my mobile and made a mental note to respond when I was by the computer.

Your name is fine